etas/oef/data/europe_parameters.json

```diff
@@ -0,0 +1,89 @@
+{
+    "oef_setting": true,
+    "name": "europe_ESHMregion_bg_alpha_bpos2",
+    "id": "europe_cut_region_bg_alpha_bpos2",
+    "fn_catalog": null,
+    "auxiliary_start": "1980-01-01 00:00:00",
+    "timewindow_start": "1990-01-01 00:00:00",
+    "timewindow_end": "2015-01-01 00:00:00",
```

Comment:
are those timewindow parameters generally not the same as in the model config input? it is confusing, if they are named the same but actually are different :D but ok